src/queries/getUserTenantsPermissions.js

```diff
@@ -8,16 +8,19 @@
  */
 const getUserTenantsPermissions = async (stripes, tenants = []) => {
   const {
-    user: { user: { id } },
     okapi: {
       url,
       token,
     }
   } = stripes;
   const userTenantIds = tenants.map(tenant => tenant.id || tenant);
 
+  const permUrl =
+    `${url}/${stripes.hasInterface('users-keycloak') ?
+      'users-keycloak' : 'bl-users'}/_self?expandPermissions=true`;
```

Comment:
Petty: This is difficult to read with the ternary split across two lines. How about 
```
const permPath = stripes.hasInterface('users-keycloak') ? 'users-keycloak' : 'bl-users';
const permUrl = `${url}/${permPath}/_self?expandPermissions=true`;
```